src/style/global-style.ts

```diff
@@ -1,6 +1,9 @@
-import { createGlobalStyle } from "styled-components";
+import { createGlobalStyle, GlobalStyleComponent } from "styled-components";
+import { ThemeObject } from "./themes/base/base-theme.config";
 
-const GlobalStyle = createGlobalStyle`
+const GlobalStyle = createGlobalStyle<
```

Comment:
This can just be as it was before, we're not passing any props into `createGlobalStyle` so this is unnecessary.

Also `GlobalStyleComponent` is what's returned by `createGlobalStyle` so this would be wrong anyway!